cluster_management/src/main/java/com/pinterest/rocksplicator/LeaderFollowerStateModelFactory.java

```diff
@@ -220,6 +230,8 @@ public void onBecomeLeaderFromFollower(Message message, NotificationContext cont
         long localSeq = Utils.getLocalLatestSequenceNumber(dbName, adminPort);
         String hostWithHighestSeq = null;
         long highestSeq = localSeq;
+        long numActiveReplicas = 1;
+        long totalReplicas = stateMap.keySet().size();
```

Comment:
I've sometimes seen helix not assign a replica to a shard. Example: {shard1: {host1:L, host2:F}. In that case, will totalReplicas be counted as 2? Is that okay?

Reply:
For now it is fine. We can discuss corner cases on this later

Reply:
I think it would be better to either remove this check, or get the total replicas using Helix API? This might cause a bug which will be hard to track later on.